pubspec.yaml

```diff
@@ -12,28 +12,28 @@ dependencies:
     sdk: flutter
 
   # From fluttercommunity.dev: Get current device information from within the Flutter application.
-  device_info_plus: ">=3.2.0 <4.2.0"
+  device_info_plus: ^6.0.0
 
   # From Dart Team: A composable, Future-based library for making HTTP requests.
-  http: ^0.13.0
+  http: ^0.13.5
 
   # From dart.dev: Platform independent access to information about the current operating system.
-  os_detect: ^2.0.0
+  os_detect: ^2.0.1
 
   # From Flutter Community: Provides an API for querying information about an application package.
-  package_info_plus: ^1.3.0
+  package_info_plus: ^2.0.0
 
   # From Flutter Team: Wraps NSUserDefaults (on iOS) and SharedPreferences (on Android).
-  shared_preferences: ^2.0.3
+  shared_preferences: ^2.0.15
 
   # From Flutter Team: A Flutter plugin for launching a URL in the mobile platform.
-  url_launcher: ^6.1.0
+  url_launcher: ^6.1.6
 
   # A dart library for comparing and incrementing version numbers with Semantic Versioning.
-  version: ">=2.0.0 <3.1.0"
+  version: ^3.0.2
 
   # A dart library for parsing, traversing, querying, transforming and building XML documents.
-  xml: ">=5.0.2 <7.0.0"
+  xml: ^6.1.0
```

Comment:
Why do you want to drop support for 5.0.2?

Reply:
Not required, just bumping the minimum from a 20-months old version (might end up being a blocker)